src/tools/msvc.jam

```diff
@@ -1624,7 +1624,7 @@ if [ MATCH (--debug-configuration) : [ modules.peek : ARGV ] ]
 .version-11.0-reg = "VisualStudio\\11.0\\Setup\\VC" ;
 .version-12.0-reg = "VisualStudio\\12.0\\Setup\\VC" ;
 .version-14.0-reg = "VisualStudio\\14.0\\Setup\\VC" ;
-.version-15.0-reg = "VisualStudio\\15.0\\Setup\\VC" ;
+.version-14.1-reg = "VisualStudio\\14.1\\Setup\\VC" ;
```

Comment:
Yes, it is better to remove them in this case. I've taken a look into #157 but I'm not sure which way to go. And I'm still not convinced that we call the thing "14.1"